LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright 2016 Yahoo Inc.
```

Comment:
this year should be 2021. it's the year this file is added

Reply:
updated year should it be VZM or Yahoo Inc. ?

Reply:
`Verizon Media`